acquia_lift.admin.wizard.inc

```diff
@@ -2353,7 +2354,13 @@ function acquia_lift_personalize_campaign_wizard_targeting_submit(&$form, &$form
   // Set the weight higher than any of the audience options.
   foreach($form_state['values']['audiences'] as $type_id => $type) {
     foreach ($form_state['values']['audiences'][$type_id] as $audience_id => $audience) {
-      if ($audience_id === ACQUIA_LIFT_TARGETING_EVERYONE_ELSE || !isset($audience['details']['weight']) || empty($audience['details']['name'])) {
+      // The machine name of the "everyone-else" audience may have changed, if
+      // tests for hte audience have changed, so it may have a suffix, like '-2'.
```

Comment:
This explanation (or something similar) might be useful at the acquia_lift_get_fallback_audience_name function since that's what most of the code uses to make this determination.  Might be worth also altering this to use that function to keep all the logic in one place (although I could see how it's clearer not to right here).